litmus-portal/okteto-dev-env-setup/README.md

```diff
@@ -76,25 +70,25 @@ This directory contains setup guide to start developing Litmus Portal on Okteto
 
-- <h4>STEP-6:</h4> Click the `Develop on Okteto` button on `env` folder's `README.md` from GitHub UI to deploy litmus-portal on Okteto cloud and start developing.
+- <h4>STEP-5:</h4> Click the `Develop on Okteto` button on `my-env` folder's `README.md` from GitHub UI to deploy litmus-portal on Okteto cloud and start developing.
   
-  [![Develop on Okteto](https://okteto.com/develop-okteto.svg)](https://cloud.okteto.com/deploy?repository=https://github.com/%GITHUB_USERNAME%/litmus&branch=dev)
+  [![Develop on Okteto](https://okteto.com/develop-okteto.svg)](https://cloud.okteto.com/deploy?repository=https://github.com/${GITHUB_USERNAME}/litmus&branch=dev)
 
-- <h4>STEP-7:</h4> Download and export the kubeconfig file from Okteto cloud settings page into `KUBECONFIG` environment variable and Login to Okteto cloud using the CLI.
+- <h4>STEP-6:</h4> Download and export the kubeconfig file from Okteto cloud settings page into `KUBECONFIG` environment variable and Login to Okteto cloud using the CLI.
 
   ```bash
   export KUBECONFIG=$HOME/Downloads/okteto-kube.config:${KUBECONFIG:-$HOME/.kube/config}
   okteto login
   ```
 
-- <h4>STEP-8:</h4> Go to specific component folders i.e. authentication, cluster-agents/subscriber, graphql-server, tools/self-deployer or frontend of `litmus-portal` folder and then run `okteto up` before manking changes to the code.
+- <h4>STEP-7:</h4> Go to specific component folders i.e. authentication, cluster-agents/subscriber, graphql-server, tools/self-deployer or frontend of `litmus-portal` folder and then run `okteto up` before manking changes to the code.
 
   ```bash
   cd litmus-portal/frontend
   okteto up
   ```
 
-- <h4>STEP-9:</h4> Start the selected component's service on its container to start developing and get the changes reflected on deployed litmus-portal component which can be accessed from Okteto UI or from `https://<service name>-<okteto-namespace>.cloud.okteto.net`. This may take several minutes to start running inside the docker container. Add the given `if block` after changing `<okteto-namespace>` with your GitHub username to `litmus-portal/frontend/src/config/index.ts` file just before the `export default` statement. Then enter username as `admin` and password as `litmus` to login from the UI.
+- <h4>STEP-8:</h4> Start the selected component's service on its container to start developing and get the changes reflected on deployed litmus-portal component which can be accessed from Okteto UI or from `https://<service name>-<okteto-namespace>.cloud.okteto.net`. This may take several minutes to start running inside the docker container. Add the given `if block` after changing `<okteto-namespace>` with your GitHub username to `litmus-portal/frontend/src/config/index.ts` file just before the `export default` statement. Then enter username as `admin` and password as `litmus` to login from the UI.
```

Comment:
The $OKTETO_NAMESPACE variable is also available inside the development container. Could we set this in the code directly, so the user doesn't have to touch any code and it just works? (I'm assuming this is about the baseURL?)

Reply:
Yeah, it is, but for this change, we have to make changes in our frontend's config file. To add Okteto as a permanent check and change API URI's dynamically. cc: @gdsoumya @rajdas98 @arkajyotiMukherjee @ajeshbaby @ksatchit